config/initializers/stripe.rb

```diff
@@ -0,0 +1,6 @@
+Rails.configuration.stripe = {
+  :publishable_key => ENV['STRIPE_PUBLISHABLE_KEY'] || Rails.application.secrets.stripe_publishable_key,
+  :secret_key      => ENV['STRIPE_SECRET_KEY'] || Rails.application.secrets.stripe_secret_key
+}
```

Comment:
In `secrets.yml` you could just do 

```
  stripe_secret_key: <%= ENV["STRIPE_SECRET_KEY"] %>
```

and then you don't need this special code.
Then we can just use `Rails.secrets` directly, forget about `Rails.configuration`, and it will work with both `secrets.yml` and ENV vars.